pkg/integration/client.go

```diff
@@ -0,0 +1,36 @@
+package integration
```

Comment:
Is this for integration tests only?  I don't see why this is named under a `pkg` dir.  I don't understand that convention.  Why wouldn't it just be `test` or `test_integration` ?

Reply:
its expected more helpers will be required over time to assist in integration. It makes it simple to import and identify.

Packages favor short single word names, test_integration would be out by default and test would be misleading

Reply:
In addition, this stuff is surfaced this way so that consumers of the API could recreate integration tests or expand on them if they wrote custom service calls with the other surfaced Client methods

Reply:
ok, it's not obvious to me what `pkg` dir is for, but after looking around I see it's a golang standard thing ;) .  I'm still a newb on this platform.

Reply:
I've never been told directly what it means, but I think  it derives from the standard library where everything in `pkg/` is inherent. 

I typically place my packages in `pkg/` in most projects, but in this one I chose `striketracker/` for the HW API-specific stuff for forwards compatibility in case wurlwind becomes a generic for multiple CDNs 

Thus `pkg/` becomes generic to wurlwind and `striketracker/` is specific to the API